draft-ietf-httpapi-ratelimit-headers.md

```diff
@@ -193,6 +193,9 @@ The term Origin is to be interpreted as described in Section 7 of {{!RFC6454}}.
 
 The "delay-seconds" rule is defined in Section 10.2.4 of {{SEMANTICS}}.
 
+This specification uses Structured Headers {{!I-D.ietf-httpbis-header-structure}} to specify syntax.
```

Comment:
Taken from https://github.com/httpwg/http-extensions/blame/master/draft-ietf-httpbis-proxy-status.md#L80

Reply:
> @ioggstream The spec we're working off of isn't called "structured headers" any more. It's called "structured field values": https://httpwg.org/http-extensions/draft-ietf-httpbis-header-structure.html (take note of the title of the document and the language within it, not the I-D's name, which is irrelevant) -- is changing back to calling it "headers" intentional?

My mistake. You can fix the original doc too ;)